helm/alfresco-content-services/values.yaml

```diff
@@ -598,6 +598,9 @@ global:
     - quay-registry-secret
 
 alfresco-sync-service:
-  enabled : true
+  syncservice:
```

Comment:
Do we need to add an extra level here?

Reply:
Yes, by default its true and just added extra level for optional to disable the service which was already there.